test/WebJobs.Script.Tests/TestScripts/CSharp/AssembliesFromSharedLocation/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿#r "..\SharedAssemblies\PrimaryDependency.dll"
```

Comment:
Yeah I'd try to go that route it if doesn't turn out to be too difficult. Seems cleaner than checking in magic binaries.